PowerEditor/src/stylers.model.xml

```diff
@@ -561,7 +561,7 @@
             <WordsStyle name="CHECKSUM_WRONG" styleID="17" fgColor="FFFF00" bgColor="FF0000" fontName="" fontStyle="0" fontSize="" />
             <WordsStyle name="GARBAGE" styleID="18" fgColor="000000" bgColor="FFFFFF" fontName="" fontStyle="2" fontSize="" />
         </LexerType>
-        <LexerType name="ini" desc="ini file" ext="">
+        <LexerType name="ini" desc="INI" ext="">

```

Comment:
INI is not a language so please restore "file" as "INI file"
https://en.wikipedia.org/wiki/INI_file